### PR TITLE
Add quick bet arrows to embedded markets

### PR DESCRIPTION
**Pull request by sipec**

### CODE REVIEW

**Review by akrolsmir — APPROVED** [review]
Looks great! Two questions:
1. Do we want to show arrows to signed-out users? We don't currently in the main site.
2. The cursor doesn't switch to a pointer on hover here; we should add `cursor-pointer` somewhere to make that work (wherever the total hitbox is for the quick bet, which I intentionally made pretty large)

File: web/pages/embed/[username]/[contractSlug].tsx [hunk 1]
```diff
@@ -114,31 +111,16 @@ function ContractEmbed(props: { contract: Contract; bets: Bet[] }) {
...
-            />
-          )}
-
-          {outcomeType === 'NUMERIC' && (
-            <NumericResolutionOrExpectation
-              contract={contract as NumericContract}
-            />
-          )}
+            )}
+            <QuickBet contract={contract} />
```
Comment (jahooma) [inline web/pages/embed/[username]/[contractSlug].tsx:121]: Binary has the bet row and quick bet? I guess that makes sense

**Review by jahooma — COMMENTED** [review]
Nice! One other suggestion is to include a screenshot in any PR with user-facing changes. Helps us understand the change, and gives an opportunity to scrutinize the change visually.

**Comment by jahooma** [comment]
One other suggestion: make the quick bet component be to the right of both the question and contract details, since the voting arrows are quite tall.

**Pull request closed without merging**